packages/drawer/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../tsconfig"
```

Comment:
I see that you build per package now. But if you ever wanted to build from a single entry point, you'll likely need ["references"](https://www.typescriptlang.org/docs/handbook/project-references.html). This will help tsc figure out the order of resolving local deps and will speed up builds. See example: https://github.com/facebook/jest/blob/bc5c3c7c49f8b190caf060319cc2d88255d9b2ff/packages/jest-cli/tsconfig.json#L7-L14